hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -761,6 +761,8 @@
       swaccess: "ro",
       hwaccess: "hwo",
       hwext: "true",
+      tags: [// Internal HW can modify status register
```

Comment:
If this is needed for these watermarks, is there any reason not to extend it to all of them?

Reply:
@martin-lueker as I guess, our strategy is to review carefully and exclude each one whenever error is reported. 
As I reviewed hjson files, some watermarks were already excluded while there are still remaining items to be reviewed.
@weicaiyang to clarify.

Reply:
yes, these 2 watermarks cause CSR test failed. Looks like the other `*_LO_WATERMARKS` are all excluded now. Not sure if `*_HI_WATERMARKS` need to be excluded too? Those seem not to affect the CSR test. Perhaps we could keep them as is?

Reply:
I was just thinking for consistency it would make sense to exclude all the watermarks.  I have no objection to doing this on an ad-hoc basis, but this is likely going to come up again.

Reply:
that sounds good to me.

Reply:
Thanks @martin-lueker.
Sounds like it's better to exclude all these watermark CSRs. @jeoongp can you also apply the exclusions for those `*_HI_WATERMARKS`

Reply:
Fixed